enterprise/coderd/groups_test.go

```diff
@@ -829,16 +837,20 @@ func TestGroup(t *testing.T) {
 		userAdminClient, _ := coderdtest.CreateAnotherUser(t, client, user.OrganizationID, rbac.RoleUserAdmin())
 		_, user1 := coderdtest.CreateAnotherUser(t, client, user.OrganizationID)
 		_, user2 := coderdtest.CreateAnotherUser(t, client, user.OrganizationID)
-
 		ctx := testutil.Context(t, testutil.WaitLong)
+
+		// nolint:gocritic // "This client is operating as the owner user" is fine in this case.
+		prebuildsUser, err := client.User(ctx, prebuilds.OwnerID.String())
```

Comment:
@dannykopping given the default hidden approach, is a reason that this test still expects the prebuilds user listed? Can I go ahead and remove it?

Reply:
The default hidden is only for user queries; I looked at this for groups too but it'd require recreating the `group_members_expanded` view.

I think it makes sense to make this visible for groups though because the user does belong to the group.

Reply:
Happy to leave this as is, since we can easily modify it later.

Reply:
based on review feedback elsewhere, we're going to add the is_system filter to these.